upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -152,7 +156,49 @@ func (c *ApplyClusterCmd) Run() error {
 	}
 	c.channel = channel
 
-	assetBuilder := assets.NewAssetBuilder(c.Cluster.Spec.Assets)
+	stageAssetsLifecycle := fi.LifecycleSync
+	securityLifecycle := fi.LifecycleSync
+	networkLifecycle := fi.LifecycleSync
+	clusterLifecycle := fi.LifecycleSync
+
+	switch c.Phase {
+	case Phase(""):
+		stageAssetsLifecycle = fi.LifecycleIgnore
+		// Everything ... the default
```

Comment:
Do you mean for this go immediately after `Phase("")`?

Reply:
This was your comment not mine ;)